test/cmdlineTests/constant_optimizer_yul/output

```diff
@@ -9,7 +9,6 @@ Optimized IR:
 object "C_12" {
     code {
         {
-            mstore(64, 128)
```

Comment:
This removal is what most of the gas savings are about...

Reply:
Will that still work with enabled stack-limit-evasion, with which this will have to become ``mstore(64, memoryguard(128))`` and be propagated properly?